source/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -318,3 +319,61 @@ it('can subtract numbers', function() {
   assert.equal(subtract(5, 12), -7, 'these numbers are equal')
 })
 ```
+
+# Watching
+
+When running in interactive mode Cypress watches the filesystem for changes to your test files. Within milliseconds of a test spec being added or updated Cypress will reload it and rerun all of the tests in that spec file.
```

Comment:
I'm hesitant to have it say 'within milliseconds', since that might not always be the case. If a user has a complicated preprocessor set up or if the test file is large, it could take seconds. I'd prefer more vague language like 'soon after'.